reaper_test.go

```diff
@@ -193,3 +185,31 @@ func Test_ReaperForNetwork(t *testing.T) {
 	assert.Equal(t, "reaperImage", provider.req.Image)
 	assert.Equal(t, "reaperImage", provider.req.ReaperImage)
 }
+
+func Test_ReaperReusedIfHealthy(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test")
+	}
```

Comment:
Do we need this separation of concerns, short tests Vs non-short tests?

Reply:
Not sure, I saw it being used in other places as well. I think if you run the Makefile `test-all` target, they don't run.
But happy to remove it as well.

Reply:
Mmmm I see them being used in the example modules for nginx, redis and cocroachdb 🤔 Probably worth it to remove them all in a follow-up PR

Reply:
As all checks are succeeding, please let me know whether to remove this now, or you will handle it in a separate PR.
Because then I believe this PR is good to merge?

Reply:
Let's remove this here as it's completely new, and the rest in a follow up PR 🙏 

Once there, I think we are ready to go